mne/bem.py

```diff
@@ -1655,6 +1683,9 @@ def make_flash_bem(subject, overwrite=False, show=True, subjects_dir=None,
         Show surfaces to visually inspect all three BEM surfaces (recommended).
     subjects_dir : string, or None
         Path to SUBJECTS_DIR if it is not set in the environment.
+    flash_path : str | None
+        Path to the flash images. If None (default), the current folder is
+        used.
```

Comment:
This was actually using the current folder before. Now we could  default to the correct place, but is ``mri/bem/flash`` really the correct folder and not ``/mri/flash/parameter_maps``? I can't find ``mri/bem/flash`` anywhere on my data.